java-extension/src/main/java/com/microsoft/vscode/spring/boot/dashboard/java/extension/JmxService.java

```diff
@@ -12,15 +14,42 @@
 
 public class JmxService {
 
+	private static final String ADMIN_DEFAULT_OBJECT_NAME = "org.springframework.boot:type=Admin,name=SpringApplication";
+
+	private static String getPropertyViaAdmin(MBeanServerConnection connection, String propertyName, Class<?> propertyType) throws Exception {
+		try{
+			final ObjectName objectName = new ObjectName(ADMIN_DEFAULT_OBJECT_NAME);
+
+			Object propertyObj = connection.invoke(
+					objectName,
+					"getProperty",
+					new String[] {propertyName },
+					new String[] {propertyType.getName() }
+
+			);
+
+			return Objects.toString(propertyObj, null);
+
+		}catch(InstanceNotFoundException ex){
+			return null;
+		}
+	}
+
 	static String getPortViaAdmin(MBeanServerConnection connection) throws Exception {
 		try {
 			String DEFAULT_OBJECT_NAME = "org.springframework.boot:type=Admin,name=SpringApplication";
 			ObjectName objectName = new ObjectName(DEFAULT_OBJECT_NAME);
```

Comment:
can reuse `ADMIN_DEFAULT_OBJECT_NAME`

Reply:
or remove this method if not used any more.

Reply:
I forgot to remove the original `getPortViaAdmin` method